src/utils/is-json-schema.ts

```diff
@@ -0,0 +1,14 @@
+import * as Lodash from 'lodash';
+import { TJSONSchema } from '../types/json-schema';
+
+export function isJSONSchema(obj: {}): obj is TJSONSchema {
```

Comment:
It's kinda tricky to check wether an object is a JSON schema or not, given that `{}` is a valid JSON schema. I wanted to add this helper originally but then thought I couldn't make it accurate.

Can you share the use case that led you to this addition?

Reply:
Something like `@body(oneOf([schemaA, schemaB])` will fail.